x-pack/plugins/index_management/public/application/sections/home/index_list/index_table/index_table.js

```diff
@@ -97,17 +100,14 @@ export class IndexTable extends Component {
 
   componentDidMount() {
     this.props.loadIndices();
-    this.interval = setInterval(this.props.reloadIndices, REFRESH_RATE_INDEX_LIST);
-    const {
-      filterChanged,
-      filterFromURI,
-      showHiddenIndicesChanged,
-      showHiddenIndices,
-      location,
-    } = this.props;
-
-    if (filterFromURI) {
-      const decodedFilter = decodeURIComponent(filterFromURI);
+    this.interval = setInterval(
```

Comment:
Just a note for our future selves: this is the code we want 4 (or more!) 👀 on when we get flakiness in our tests 😊

Reply:
Hi! I am the future self 😊 : don't worry, it's okay, there is `jest. useFakeTimers()` that gives us control over this.